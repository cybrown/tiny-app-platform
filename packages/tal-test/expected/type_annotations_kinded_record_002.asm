main()
  entry:
    FunctionRef       name: Debug_0
    DeclareLocal      name: Debug, mutable: false, hasInitialValue: true
    Pop               inBlock: false
    FunctionRef       name: func_1
Debug_0(value)
  entry:
    Literal           0
    MakeRecord
func_1()
  entry:
    Local             Debug
    Literal           0
    MakeArray
    Literal           "value"
    Local             a
    Literal           1
    MakeRecord
    Kinded
