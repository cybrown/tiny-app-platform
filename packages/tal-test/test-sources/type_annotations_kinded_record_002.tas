fun Debug(value: number) {

}

Debug { value: a }
