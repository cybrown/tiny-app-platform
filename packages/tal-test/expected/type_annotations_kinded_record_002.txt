(5:16) Unknown symbol: a
kinded-record
